src/main/java/com/revature/models/CarDTO.java

```diff
@@ -0,0 +1,26 @@
+package com.revature.models;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@Data
+@NoArgsConstructor
+public class CarDTO {
+    private int carId;
```

Comment:
Should check all DTOs for spaces vs tabs. Either tabs or spaces would work, but since this repository is using tabs in the large majority of locations, new code should stay consistent.

Reply:
Resolved in 325399c